test/quantization/core/test_quantized_op.py

```diff
@@ -3479,6 +3482,33 @@ def test_embedding_bag_4bit(self, num_embeddings,
                                                sparsity=sparsity,
                                                atol=0.1, rtol=1e-2)
 
+    """ Tests the correctness of the embedding_bag_2bit quantized operator """
+    @given(num_embeddings=st.integers(10, 100),
+           embedding_dim=st.integers(5, 50).filter(lambda x: x % 4 == 0),
+           num_offsets=st.integers(1, 20),
+           use_32bit_indices=st.booleans(),
+           use_32bit_offsets=st.booleans(),
+           enable_per_sample_weights=st.booleans(),
+           include_last_offset=st.booleans(),
+           fallback_to_no_sparse=st.booleans(),
+           sparsity=st.sampled_from([0.0, 0.5, 0.7]))
+    def test_embedding_bag_2bit(self, num_embeddings,
```

Comment:
Are we also testing FP16 inputs in this test?

Reply:
FP16 makes sense for prepack/unpack? which is covered in 

```
 """ Tests the correctness of the embedding_bag_8bit pack/unpack op against C2 """
    @given(num_embeddings=st.integers(10, 100),
           embedding_dim=st.integers(5, 50).filter(lambda x: x % 4 == 0),
           num_batches=st.integers(1, 5),
           data_type=st.sampled_from([np.float32, np.float16]),)
    def test_embedding_bag_byte_unpack(self, num_embeddings, embedding_dim, num_batches, data_type):
        pack_fn = torch.ops.quantized.embedding_bag_byte_prepack
        unpack_fn = torch.ops.quantized.embedding_bag_byte_unpack

        self._test_embedding_bag_unpack_fn(
            pack_fn, unpack_fn, num_embeddings, embedding_dim, 8, False, num_batches, data_type=data_type)
```